src/osm_configurator/model/project/config_phase_enum.py

```diff
@@ -24,3 +24,9 @@ def get_name(self):
             str: Name of the Phase.
         """
         return self.value
+
+    def equals(phase: str) -> ConfigPhase | None:
```

Comment:
better name or descirbe in pydoc what this method does.
A "normal" equals method would return a boolean.